pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -290,6 +290,12 @@ func (facts *PipelineRunFacts) IsRunning() bool {
 	return false
 }
 
+// IsCancelled returns true if the PipelineRun won't be scheduling any new Task because it was cancelled
+func (facts *PipelineRunFacts) IsCancelled() bool {
```

Comment:
Sorry for being very picky here, please update the comment to not include the scheduling aspect but focusing on what this function does.

Reply:
I notice the other functions have the similar style of comments. Since you are cleaning up, do you mind updating those functions (`IsGracefullyStopped` and `IsGracefullyCancelled`) as well. Its good to have additional context around scheduling but this function is just comparing the status.

Reply:
this is not picky at all, I would make the same comment! Done.